assets/app/views/project.html

```diff
@@ -3,7 +3,7 @@
     <div ng-controller="OverviewController">
       <tasks></tasks>
       <div class="page-header page-header-bleed-right">
-        <h1>Project {{(project | displayName) || project.metadata.name}}</h1>
+        <h1 title="{{project.metadata.name}}">Project {{(project | displayName) || project.metadata.name}}</h1>
```

Comment:
I'm not clear why we need a `title` attribute here if it's the same value. I'm assuming @sg00dwin wanted to show the project name if the display name is different?